scripts/ci/run_integration_tests.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -eu -o pipefail
+# -e: exits if a command fails
+# -u: errors if an variable is referenced before being set
+# -o pipefail: causes a pipeline to produce a failure return code if any command errors
+
+readonly TEST_TARGETS=$(bazel query 'kind("bazel_integration_test", //examples/... union //e2e/...)')
```

Comment:
Switching in this PR from selecting based on tags to using the rule kind. Did you check that it gives the same list of targets?

Reply:
also we rely on these coming in the same order every time, might be good to comment on that, or maybe even do an explicit sort to be safe?

Reply:
yes. double checked that everything is still tested.

I'm making the assumption that bazel query is deterministic in its order. I'll add an explicit sort to be safe.